delivery/get-issue-type/action.yml

```diff
@@ -0,0 +1,65 @@
+
+name: Get issue type
+description: Returns the type of an issue
+
+inputs:
+  github_token:
+    description: 'GitHub personal API token'
+    required: true
+
+outputs:
+  result:
+    description: "Issue type"
+    value: ${{ steps.get-issue-type.outputs.result }}
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Check if an issue exists
```

Comment:
Why duplicating the code above ? isn't possible to use the same action twice ?

Reply:
I was trying to separate the concern, but yes, I could merge the two

Reply:
I merged the two actions in one.